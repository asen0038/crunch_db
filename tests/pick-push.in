set a 5 6 7
pick a 2
pluck a 2
pop a
push a 7
append a 7
get a
