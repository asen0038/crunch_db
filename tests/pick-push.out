>
ok

>
6

>
6

>
5

>
ok

>
ok

>
[7 7 7]

>
bye
